CONTRIBUTING.md

```diff
@@ -0,0 +1,101 @@
+# Contribute
+
+## Setting up
+
+1. Clone this git repository on your local machine inside a `wp-content/plugins/` directory
+2. Install [PHPUnit](https://phpunit.de/). We recommend using version 4.5, as that's the most current version that is compatible with all PHP versions we check against, and thus Travis is using that version.
+3. Activate the plugin.
+4. Visit `http://local.sitename/glotpress` to start using GlotPress. `local.sitename` is the domain where your local development WordPress site runs, dependent on configuration.
+
+## Submitting patches
+
+Whether you want to fix a bug or implement a new feature, the process is pretty much the same:
+
+0. [Search existing issues](https://github.com/GlotPress/GlotPress-WP/issues); if you can't find anything related to what you want to work on, [open a new issue](#helpful-tips-for-writing-issues).
+1. [Fork](https://github.com/GlotPress/GlotPress-WP/fork) the repository.
+2. Create a branch for each issue you'd like to address. Base your new branch on top of `develop` branch. Branches are named as such: `ISSUEID-keywords` for branches where an issue exists, or `keywords` for a branch without an issue yet. For example issue 216 for adding contributing.md would be `216-add-contributing-md`. The same branch would be named `add-contributing-md` if there wasn't an issue for it. Commit your changes.
+3. Push the code changes from your local clone to your fork.
+4. Open a pull request from your fork's feature branch to GlotPress's `develop` branch. Pull request name should be `ISSUEID-keywords`. Take a look at [one such pull request](https://github.com/GlotPress/GlotPress-WP/pull/241) to get an idea of it.
+
+We use the [Git workflow](https://github.com/GlotPress/GlotPress-WP/wiki/5.-Git-workflow). Please have a read through, as it will make everyone's life easier. Should you have questions about that, reach out to us on Slack (see below).
+
+Pull request should have `Fixes #issue` / `Closes #issue`, or, if it's a partial solution, `Part of #issue` in the message to [enable automatic issue closing when a PR is merged](https://help.github.com/articles/closing-issues-via-commit-messages/).
+
+It doesn't matter if the code isn't perfect. The idea is to get it reviewed early and iterate on it.
+
+Lastly, please follow the [WordPress Coding Standards](http://make.wordpress.org/core/handbook/coding-standards/).
+
+## Testing patches / PRs
+
+1. Check out the branch the PR was submitted on.
+2. Leave feedback on the PR. Write up what you tested, what your results were and whether it solves the issue or not.
+
+If you need help with checking out the branch, or the PR was submitted from someone's fork, GitHub provides help in checking out the correct branch. Navigate to the PR in question, and you should see a link to command line instructions. Follow those instructions.
+
+![](https://cloud.githubusercontent.com/assets/617637/12953405/9df99808-d01a-11e5-88eb-958d23871f87.png)
+
+## Running and writing tests
+
```

Comment:
Add: 
GlotPress includes automated test code but coverage is not complete at this time.  Contributing new tests for GlotPress is a great way to better understand the codebase and contribute to the project at the same time.